lib/helper.rb

```diff
@@ -21,24 +21,18 @@ def self.is_int?(string)
   end
 
   def self.content_is_binary?(content)
-    # Code by http://www.thecodingforums.com/threads/test-if-file-is-binary.843447/#post-4572282
-    # Modified by SUSE Linux GmbH
     ascii = 0
-    control = 0
     binary = 0
 
     content.each_byte do |byte|
-      case byte
-      when 0...32
-        control += 1
-      when 32...128
+      if (32..128).include?(byte)
         ascii += 1
       else
-        binary += 1
+        # count binary but exclude newlines and tabs
+        binary += 1 unless [9, 10, 13].include?(byte)
       end
```

Comment:
Why would you treat newlines and tabs differently than other printable characters? I would go with a simple

```ruby
printable = 0

content.each_byte do |byte|
  printable += 1 if [9,10,13].include?(byte) || (32..128).include?(byte)
end

printable.to_f / content.size < 0.9
```

or something like that.